### ISSUE TITLE
Print minimal imports

### ISSUE THREAD

**Issue by chshersh** [kowainik/smuggler#42]
Basically conversion from implicit imports to explicit imports.

Requires contribution to GHC:

* https://ghc.haskell.org/trac/ghc/ticket/15439

**Comment by kahlil29** [comment]
It looks like the GHC issue has been fixed 👍

**Comment by chshersh** [comment]
@kahlil29 Yes, it's actually @vrom911 who contributed this patch the GHC and now this function is reexported in GHC 8.6.1, so we can use it! But I propose to think about UX for this issue first.
